src/lib/rest/rest.cpp

```diff
@@ -922,14 +924,19 @@ static ApiVersion apiVersionGet(const char* path)
     return V2;
   }
 
-  // Different from v2, v1 is case-insensitive (see case/2057 test)
```

Comment:
In that function we know whether the uri path was /nsgi9 or /ngsi10 etc. That info is 'thrown away' and we keep only 'v1', etc.
To not repeat code we could create an enum for that info. Put a new var in ConnectionInfo